.gitattributes

```diff
@@ -0,0 +1,81 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Files with these extensions are accepted source files inside the ReactOS
+# tree. They should have native line endings of the OS.
+*.acf text
+*.asm text
+*.c text
+*.cc text
+*.cmake text
+*.cpp text
+*.cxx text
+*.h text
+*.hpp text
+*.hxx text
+*.idl text
+*.inc text
+*.inl text
+*.klc text
+*.l text
+*.lds text
+*.md text
+*.rc text
+*.s text
+*.S text
+*.sfd text
+*.spec text
+*.svg text
+*.txt text
+*.TXT text
+*.y text
+
+# Files with these extensions end up in the built ReactOS system, so they
+# need to have CRLF line endings.
+*.bat text eol=crlf
+*.cmd text eol=crlf
+*.exp text eol=crlf
+*.hhc text eol=crlf
+*.hhp text eol=crlf
+*.ini text eol=crlf
+*.INI text eol=crlf
+*.inf text eol=crlf
+*.inf.tpl text eol=crlf
+*.js text eol=crlf
+*.manifest text eol=crlf
+*.mc text eol=crlf
+*.mof text eol=crlf
+*.rgs text eol=crlf
+*.sif text eol=crlf
+*.vbs text eol=crlf
+
+# Files with these extensions are accepted binary files inside the ReactOS
+# tree. Git must not tamper with them at all!
+*.avi binary
+*.bin binary
+*.bmf binary
+*.bmp binary
+*.BMP binary
+*.chm binary
+*.cur binary
+*.dll binary
+*.emf binary
+*.gif binary
+*.ico binary
+*.jpg binary
+*.mp3 binary
+*.nls binary
+*.otf binary
+*.pdn binary
+*.pfb binary
+*.pfm binary
+*.png binary
+*.psd binary
+*.ttc binary
+*.ttf binary
+*.wav binary
+*.xcf binary
+
+# All other extensions not explicitly mentioned here are left for Git to
+# handle automatically.
+# You must not rely on them having a particular line ending style!
```

Comment:
I know that in @ThFabba by default we marked them as binary, and then overrided these via the extension settings.

Reply:
See https://gist.github.com/ThFabba/3b2a95b068f85a6683e6f48a978eea0c#gistcomment-2220363

Reply:
Thanks for the reference.